animad-service-administration/animad-administration-service/src/main/java/de/muenchen/service/QueryServiceChanged.java

```diff
@@ -0,0 +1,77 @@
+
+package de.muenchen.service;
+
+import java.util.*;
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import javax.transaction.Transactional;
+
+import org.apache.lucene.index.Term;
+import org.apache.lucene.search.Query;
+import org.apache.lucene.search.TermQuery;
+import org.hibernate.search.jpa.FullTextEntityManager;
+import org.hibernate.search.jpa.FullTextQuery;
+import org.hibernate.search.jpa.Search;
+import org.hibernate.search.query.dsl.BooleanJunction;
+import org.hibernate.search.query.dsl.QueryBuilder;
+import org.springframework.stereotype.Repository;
+
+@Repository
+@Transactional
+public class QueryServiceChanged {
```

Comment:
*Changed könnte man fast alles nennen, an dem man arbeitet :)

Reply:
Changed heißt es nur, weil die Klassen gibt's ja schon, aber in dem Projekt eben nur in einer lib. Eigentlich soll der Code in QueryService mit rein.